admin.php

```diff
@@ -108,30 +110,54 @@ function output_page_header( $id, $messages = array() ) {
 ?>
 
 <div class="wrap">
-	<h2 id="edit-site">
+	<h1 id="edit-site">
```

Comment:
The h tags that code changes a while ago, making this look broken, another cheeky styling fix.